src/Exception/ExceptionInterface.php

```diff
@@ -4,9 +4,11 @@
 
 namespace Mezzio\Authentication\Exception;
 
+use Throwable;
+
 /**
  * Marker interface for package-specific exceptions
  */
-interface ExceptionInterface
+interface ExceptionInterface extends Throwable
```

Comment:
While there is a BC break, this interface is supposed to be implemented on `Throwable` types anyway